src/integration/data/delete.test.ts

```diff
@@ -49,39 +49,39 @@ describe('delete', () => {
     await ns.upsert(recordToUpsert);
 
     // Await record freshness, and check record upserted
-    let stats = await waitUntilRecordsReady(ns, namespace, recordIds);
+    const stats = await waitUntilRecordsReady(ns, namespace, recordIds);
     if (stats.namespaces) {
       expect(stats.namespaces[namespace].recordCount).toEqual(1);
     } else {
       fail('Expected namespaces to be defined');
     }
 
     // Look more closely at one of the records to make sure values set
-    const fetchResult = await ns.fetch(['0']);
-    const records = fetchResult.records;
-    if (records) {
-      expect(records['0'].id).toEqual('0');
-      expect(records['0'].values.length).toEqual(5);
-    } else {
-      fail(
-        'Did not find expected records. Fetch result was ' +
-          JSON.stringify(fetchResult)
-      );
-    }
+    const fetchAssertions = [
```

Comment:
We were seeing a lot of flake in the delete tests across various scenarios. Removed the sleeps and went with `assertWithRetries` which seems to help a lot here. This test probably needs some cleanup eventually.